libs/platform/approval-flow/approval-flow.component.ts

```diff
@@ -896,6 +902,10 @@ export class ApprovalFlowComponent implements OnInit, OnChanges, OnDestroy {
         this._approvalProcess = approvalProcess;
         this._graph = generateApprovalFlowGraph(this._approvalProcess.nodes);
 
+        if (this.removeSpaceNode) {
+            this._graph.columns = this.removeSpaceNodesFromColumns(this._graph.columns);
```

Comment:
`this._graph.columns` can never be `undefined` as it has been initialized as an empty array

![Oscar ScreenCapture 2024-03-13 at 16 16 2](https://github.com/SAP/fundamental-ngx/assets/7035787/ef61eb69-0538-4252-9ec6-49888e2f3dbd)